File: 1929_Concatenation_of_Array.js
var getConcatenation = function(nums){
    let ans = new Array(2*nums.length)
    for(let i = 0 ; i < nums.length ; i++){
        ans[i] = nums[i]
        ans[i+nums.length] = nums[i]
    }
    return ans 
}
let nums = [1,2,1]
console.log(getConcatenation(nums));

File: 1929_ConcatenationofArray.java
class Solution {
    public int[] getConcatenation(int[] nums) {
       int n = nums.length;
       int[] concatArray = new int[n*2];
       for(int i = 0; i < n ; i++){
           concatArray[i] = nums[i];
           concatArray[n+i] = nums[i];
       }
       return concatArray;
    }
}